token-metadata/program/src/utils.rs

```diff
@@ -963,6 +973,19 @@ pub fn process_create_metadata_accounts_logic(
 
     let mut metadata: Metadata = Metadata::from_account_info(metadata_account_info)?;
     let compatible_data = data.to_v1();
+
+    // This allows the Bubblegum program to create metadata with verified creators since they were
+    // verified already by the Bubblegum program.
+    const BUBBLEGUM_ACTIVATED: bool = false;
+    let allow_direct_creator_writes = if BUBBLEGUM_ACTIVATED
+        && mint_authority_info.key == &BUBBLEGUM_SEED_AUTHORITY
```

Comment:
This actually won't work as-is, because the Bubblegum program ID will not be the `mint_authority` and the program cannot be a signer.  @samuelvanderwaal noticed this as we reviewed it offline.

We also cannot just hardcode the mint_authority PDA information because `mint_authority` has a dynamic seed based on the mint account, which itself has dynamic seeds.

Sam came up with a potential solution that is simple and elegant in that it will not require changing the structure of the PDAs in Bubblegum:
1. Change the name `BUBBLEGUM_SEED_AUTHORITY` to `BUBBLEGUM_PROGRAM_ADDRESS` because that is what it is.
2. The `mint_authority` is still a signer, and since it is a PDA of Bubblegum, instead verify that the `mint_authority_info` is owned by the Bubblegum program.

Reply:
Fixed in latest version